### DISCUSSION TITLE
Github Pages

### DISCUSSION THREAD

**Discussion by IHateYourCode** [texstudio-org/texstudio#1806] [Ideas]
I was wondering if you have considered using **Github Pages**
for the wiki, instead of the default wiki tab.

This would enable to have **subfolders for better separation** of content.

### Example
*The current compilation page could be split*
*up into a general dependencies page and sub-pages*
*specific to the OS*

### Recommend
I would recommend to then simply move the wiki files to a `Wiki` branch
(or even a separate repo) and put a link to the **io** page in the **README**.

A separate repo seems more sensible in this case though, considering
**texstudio-org** isn't stuffed with hundreds of repos and branches are
less visible and accessible compared to using a dedicated repository.

### Theme
The wiki would also be styled, thanks to the **Github Pages** themes.

### This Would Be A Great Use For
the `Scripts.md` page as well, considering it **spans over `1.2k` lines**!

### Demo
If this is considered `too much work` or a **demonstration** is wanted,
***I would be willing to create such a repo*** and later, if it is deemed wanted,
***transfer it*** over to **texstudio-org**.

**Comment by sunderme** [comment]
in principle I like the easy access to change the wiki. It is considerably easier to adapt than any git content.
And there is still hope that users leave some tips&tricks, etc.

**Reply by IHateYourCode** [reply]
Well in that regard it would be possible to combine the two,
use github pages for **stable content**, while allowing for
*easy edits*, for things like scripts, with the **wiki** functionality.

**Comment by sunderme** [comment]
what would you consider as stable content? I don't think that there is much (apart from Contributing)

**Reply by IHateYourCode** [reply]
`Translating.md` `Reporting-Bugs.md` ,..
In general things that are already there.

I'm suggesting to use the wiki as a forum for **new** things,
if it's **scripts**, **tips**, .. that can be worked on there until
usable and then adapted to the `stable` content.

If someone has worked on a new script for example, to the
point where it's properly documented and the like, they will
probably want to version control / make changes visible
anyways.

Actually, for that matter, if the Wiki was a separate repo,
one could use the `Issues` / `Discussions` there as exactly
that, a place where **fixes** / **additions** could be
discussed easily, this would work even better than the wiki.

**Reply by sunderme** [reply]
I still like the wiki for scripts,tips, compiling and FAQ. 
This these things need adaptation regularly.

**Reply by sunderme** [reply]
And then I am not really convinced that we gain much to put the rest in a github pages. It is easy enough for anyone to contribute on those if they like to.

**Reply by IHateYourCode** [reply]
[Fuide]: https://github.com/Fuide

Mhmh, well, how about we go with this,
I will just create it in the [Fuide]s org and
once it's done, you can take a look and
decide if you still aren't convinced.

If you aren't after that, I'll just maintain
it separately and if you are, I can tranfer it.

**Reply by sunderme** [reply]
sure, why not
